app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if (isinstance(other, Distance)):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if (isinstance(other, Distance)):
+            self.km = self.km + other.km
+        else:
+            self.km = self.km + other
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        self.km = self.km * other
+        return self
+
+    def __truediv__(self, other: int | float) -> Distance:
+        self.km = round((self.km / other), 2)
+        return self
```

Comment:
`other` can not be of type `Distance` in this method - make a check for this
also, this methods should return a new instance of Distance class